pkg/scalers/azure/azure_eventhub_checkpoint.go

```diff
@@ -224,8 +224,22 @@ func (checkpointer *defaultCheckpointer) extractCheckpoint(get *azblob.DownloadR
 }
 
 func getCheckpoint(ctx context.Context, httpClient util.HTTPDoer, info EventHubInfo, checkpointer checkpointer) (Checkpoint, error) {
+	var podIdentity = info.PodIdentity
+
+	// For back-compat, prefer a connection string over pod identity when present
+	if len(info.StorageConnection) != 0 {
+		podIdentity.Provider = kedav1alpha1.PodIdentityProviderNone
+	}
+
+	if podIdentity.Provider == kedav1alpha1.PodIdentityProviderAzure || podIdentity.Provider == kedav1alpha1.PodIdentityProviderAzureWorkload {
```

Comment:
What if somebody still wants to specify a connection string for Blob Storage? We should allow that because this is a breaking change.

However, if `storageAccountName` is not configured but Pod Identity is used, it would be good to log a warning.

Reply:
@tomkerkhove, can we close on this comment?